lib/ansible/modules/subversion.py

```diff
@@ -24,17 +24,20 @@
   repo:
     description:
       - The subversion URL to the repository.
+    type: str
     required: true
     aliases: [ name, repository ]
   dest:
     description:
       - Absolute path where the repository should be deployed.
-    required: true
+      - The destination directory must be specified unless checkout=no, update=no, and export=no.
```

Comment:
```suggestion
      - The destination directory must be specified unless I(checkout=no), I(update=no), and I(export=no).
```

Reply:
Fixed in https://github.com/ansible/ansible/pull/70319/commits/65503c2b7a98843dbf867b8a9fd90d2f7b43c6e1.